tests/LSP8IdentifiableDigitalAsset/extensions/LSP8Enumerable.behavior.ts

```diff
@@ -0,0 +1,80 @@
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { ethers } from "hardhat";
+import { LSP8EnumerableTester } from "../../../types";
+
+export type LSP8EnumerableTestAccounts = {
+  owner: SignerWithAddress;
+  tokenReceiver: SignerWithAddress;
+};
+
+export const getNamedAccounts = async (): Promise<LSP8EnumerableTestAccounts> => {
+  const [owner, tokenReceiver] = await ethers.getSigners();
+  return { owner, tokenReceiver };
+};
+
+export type LSP8EnumerableDeployParams = {
+  name: string;
+  symbol: string;
+  newOwner: string;
+};
+
+export type LSP8EnumerableTestContext = {
+  accounts: LSP8EnumerableTestAccounts;
+  lsp8Enumerable: LSP8EnumerableTester;
+  deployParams: LSP8EnumerableDeployParams;
+};
+
+export const shouldBehaveLikeLSP8Enumerable = (
+  buildContext: () => Promise<LSP8EnumerableTestContext>
+) => {
+  let context: LSP8EnumerableTestContext;
+
+  beforeAll(async () => {
```

Comment:
I would suggest replacing this `beforeAll()`, by a `beforeEach()`.

```suggestion
  beforeEach(async () => {
```

To avoid having to write at the end of each tests.

```ts
await context.lsp8Enumerable.burn(tokenId);
```

Unless there is a specific reason that I am not aware of? (like the test suite running faster, but since it is running in the CI, it should not be a big issue).

@YamenMerhi @jakeprins @Hugoo  what do you think about this? :)